{{ cookiecutter.repo_name }}/.gitignore

```diff
@@ -1,5 +1,5 @@
-# Created by https://www.gitignore.io/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks
-# Edit at https://www.gitignore.io/?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks
+# Created by https://www.toptal.com/developers/gitignore/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks,macos,linux,windows
+# Edit at https://www.toptal.com/developers/gitignore?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks,macos,linux,windows
```

Comment:
```suggestion
# Created by https://www.toptal.com/developers/gitignore/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,visualstudiocode,jupyternotebooks,macos,linux,windows
# Edit at https://www.toptal.com/developers/gitignore?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,visualstudiocode,jupyternotebooks,macos,linux,windows
```

Same as other higher-level .gitignore.

Reply:
Made the change in [`89970d8`](https://github.com/ukgovdatascience/govcookiecutter/pull/12/commits/89970d8f007a4df08a82f53d26dafd77ecc1d3d0).